lib/experimental/class-wp-webfonts.php

```diff
@@ -121,7 +121,7 @@ public function get_providers() {
 	 * @since 6.0.0
 	 *
 	 * @param array $webfont Webfont to be registered.
-	 * @return bool True if successfully registered, else false.
+	 * @return string|bool The font family slug if successfully registered, else false.
```

Comment:
This will no longer return `true`. So the `@return` needs to be `string|false`.

```suggestion
	 * @return string|false The font family slug if successfully registered, else false.
```